crates/config/src/lib.rs

```diff
@@ -387,6 +388,9 @@ pub struct Config {
     /// Address labels
     pub labels: HashMap<Address, String>,
 
+    /// Soldeer dependencies
+    pub dependencies: Table,
```

Comment:
can we make this more strongly typed? see e.g https://github.com/foundry-rs/foundry/pull/2386/files for some inspiration

Reply:
solved